PowerUp/app/src/main/res/layout-land/dressing_room.xml

```diff
@@ -5,23 +5,53 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@drawable/curtains">
-    <ImageView
-        android:layout_width="150dp"
-        android:layout_height="200dp"
-        android:src="@drawable/character"
-        android:id="@+id/characterImageView"
-        android:layout_alignBottom="@+id/health_text_ImageView"
-        android:layout_toRightOf="@+id/ageImageView"
-        android:layout_toEndOf="@+id/ageImageView" />
+
+    <RelativeLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:layout_toRightOf="@+id/karmaImageView"
+        android:layout_toEndOf="@+id/karmaImageView"
+        >
+        <ImageView
+            android:id="@+id/faceView"
+            android:layout_width="@dimen/face_view_width"
+            android:layout_height="@dimen/face_view_height"
+            android:layout_centerHorizontal="true"
+            android:src="@drawable/face1" />
+
+        <ImageView
+            android:id="@+id/eyeView"
+            android:layout_width="@dimen/eye_view_width"
+            android:layout_height="@dimen/eye_view_height"
+            android:layout_alignBottom="@id/faceView"
+            android:layout_centerHorizontal="true"
+            android:layout_marginBottom="@dimen/eye_view_margin_bottom"
+            android:src="@drawable/eye1" />
+
+        <ImageView
+            android:id="@+id/hairView"
+            android:layout_width="@dimen/hair_view_width"
+            android:layout_height="@dimen/hair_view_height"
+            android:src="@drawable/hair2"
+            android:layout_centerHorizontal="true" />
+
+        <ImageView
+            android:id="@+id/clothView"
+            android:layout_width="@dimen/cloth_view_width"
+            android:layout_height="@dimen/cloth_view_height"
+            android:src="@drawable/cloth1"
+            android:layout_alignTop="@+id/eyeView"
+            android:layout_centerHorizontal="true" />
+    </RelativeLayout>
 
     <ImageView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:id="@+id/karmaImageView"
         android:src="@drawable/karma"
-        android:layout_alignParentTop="true"
-        android:layout_toLeftOf="@+id/characterImageView"
-        android:layout_toStartOf="@+id/characterImageView" />
+        android:layout_alignParentTop="true" />
+
```

Comment:
remove extra lines. Its better to run Android Studio's code formatter before submitting code.